src/components/pages/Notes/Todo/TodoItem.js

```diff
@@ -11,17 +11,33 @@ const TodoItem = props => {
         <Row>
           <Col xs='auto' className='ml-1 pr-0'>
             <Button
-              variant='success'
-              onClick={() => checkTask(task.id, !task.checked)}
-              className='button-shadow'>
-              <i className={`fas fa-${task.checked ? 'check' : 'times'} fa-fw`} />
+              variant=''
```

Comment:
This line is redundant.

Reply:
without this line, border of the button get blue tint (becouse primary variant is default)

Reply:
Alright, fair enough. I suppose it's the easiest way of fixing that problem. :D